tests/Controller/WebhookControllerTest.php

```diff
@@ -58,6 +58,11 @@ public function getTests()
                 'pull_request.opened.json',
                 ['pull_request' => 3, 'status_change' => 'needs_review', 'pr_labels' => ['Bug']],
             ],
+            'On pull request opened with target branch' => [
+                'pull_request',
+                'pull_request.opened_target_branch.json',
+                ['pull_request' => 5, 'status_change' => 'needs_review', 'pr_labels' => ['Bug'], 'milestone'=>'4.4'],
```

Comment:
How can I test this properly?

Reply:
Not really a great way - it's a very basic "smoke test" :/